lib/modules/datasource/galaxy-collection/index.ts

```diff
@@ -36,7 +36,7 @@ export class GalaxyCollectionDatasource extends Datasource {
     const [namespace, projectName] = packageName.split('.');
 
     // TODO: types (#22198)
-    const baseUrl = `${registryUrl}api/v2/collections/${namespace}/${projectName}/`;
+    const baseUrl = `${registryUrl}api/v3/collections/${namespace}/${projectName}/`;
```

Comment:
Are you sure this is a `fix` type PR? Switching to a new API _may_ be a breaking change.

I guess at the very least this kind of change should go in a `feat:` type PR, so it's not mingled with "real patches"?

Reply:
I agree this should be a `feat`.

Also is v3 really returning the exact same response on the same sub path?

Reply:
Thank you for your feedback. Will change it to feat.
@secustor I've started to check it and the response is probably different. I don't know typescript enough to understand the errors it throws.
In any case, the v2 version is broken.